app/models/course.rb

```diff
@@ -338,4 +342,13 @@ def set_defaults
   def remove_unnecessary_enrichments_validation_message
     self.errors.delete :enrichments if self.errors[:enrichments] == ['is invalid']
   end
+
+  def validate_course_syncable
+    if findable?.blank?
```

Comment:
dont think you need blank as `findable?` is a bool right?

Reply:
Discussed. We need to check for false rather than true. It will be left how it is now.